torch/_inductor/codegen/cuda/cuda_kernel.py

```diff
@@ -19,23 +23,26 @@ def _normalize_idx(index: int, total_length: int) -> int:
 
 class CUDAKernel(Kernel):
     """
-    Kernels defined by C++ CUDA.
+    Baseclass for CUDA / Cutlass based Kernels
     """
 
     overrides = OpOverrides  # type: ignore[assignment]
 
 
 class CUDATemplateKernel(CUDAKernel):
     """
-    Template kernels defined by C++ CUDA.
+    Template kernels defined by CUDA / Cutlass in C++.
     """
 
     _EXTRA_CPP_ARGS = "size_t* workspace_size, uint8_t* workspace, cudaStream_t stream"
 
-    def __init__(
-        self,
-        kernel_name,
-    ):
+    def __init__(self, kernel_name):
+        """
+        Initializes a new instance of the CUDATemplateKernel class.
+
+        Args:
+            kernel_name (str): The name of the kernel.
+        """
```

Comment:
doc changes are great, but would be easier to review if it was a separate PR

Reply:
makes sense, I would propose I separate it out next time, instead of splitting it in retrospective..